test_scripts/Polices/Validation_of_PolicyTables/248_ATF_pt_snapshot_path_is_incorrect.lua

```diff
@@ -89,26 +91,27 @@ function Test:Precondition_StopSDL()
 end
 
 function Test:Precondition_ReloadNewINIFile()
-  oldPathToPtSnapshot = self.changePtsPathInSdlIni(INCORRECT_LINUX_PATH_TO_POLICY_SNAPSHOT_FILE)
+  self.changePtsPathInSdlIni(INCORRECT_LINUX_PATH_TO_POLICY_SNAPSHOT_FILE)
 end
 
 --[[ Test ]]
 commonFunctions:newTestCasesGroup("Test")
+function Test.TestStep_start_sdl()
+  StartSDL(config.pathToSDL, config.ExitOnCrash)
+  os.execute("sleep 5")
+end
 
 function Test:TestStep_VerifySDL_Stops()
-  --TODO(istoimenova): Should be checked when ATF problem is fixed with SDL crash
-  EXPECT_HMINOTIFICATION("BasicCommunication.OnSDLClose")
-  local result = testCasesForPolicySDLErrorsStops:CheckSDLShutdown(self)
-  if (result == false) then
-    self:FailTestCase("Error: SDL doesn't stop.")
-  end
+  testCasesForExternalUCS.checkSDLStatus(self, sdl.STOPPED)
 end
 
-function Test:TestStep_CheckSDLLogError()
-  local result = testCasesForPolicySDLErrorsStops.ReadSpecificMessage("PathToSnapshot is not correct")
-  if (result == false) then
-    self:FailTestCase("Error: message 'PathToSnapshot is not correct' is not observed in smartDeviceLink.log.")
-  end
+function Test.TestStep_CheckSDLLogError()
```

Comment:
@dboltovskyi, I remove the whole step, not to have misunderstand because:
1. SDL immediately stops with wrong path. It doesn't return OnSDLClose, because not started at all.
2. Error for path is not always printed in log because it's the last message.

See e20c479eeded5f509273215ad0cd0ba7404c8355